Issue 134

Dechechi opened the pull request.

Commit:
Issue-134 : JavaDoc sem tags h1, tentativa de arrumar problema com codacy e uso do NoSY

Commit:
Issue-134 Alteração do ProfessorJpaController.java para melhoria dos java docs, correção ao codacy e uso do NoSY

Commit:
Issue-134 Arrumando algumas questões do código para melhor avaliação do
codacy

Alterações realizadas na classe ProfessorJpaController

calebepb commented on src/datamapper/ProfessorJpaController.java:
```diff
@@ -42,18 +67,26 @@ public void create(Professor professor) {
+     * @param professor Object
+     * @throws NonexistentEntityException
+     */
+    public void edit(Professor professor) throws NonexistentEntityException {
```
Tanto o parâmetro `professor` quanto a exceção `NonexistentEntityException` precisam de uma documentação.

calebepb commented on src/datamapper/ProfessorJpaController.java:
```diff
@@ -13,25 +14,49 @@
 public class ProfessorJpaController implements Serializable {
 
+    /**
+     * <p>Atribute of the class ProfessorJpaController that references to an EntityManagerFactory, used all over the class
```
Arrumar!

calebepb commented:
Alum disso, sugiro fazer um rebase e arrumar os conflitos que existem.

Commit:
Issue-134
Revisados alguns conceitos recomendados pelo codacy e corrigindo problemas
com conflitos no merge

Commit:
Issue-134 - Corrigindo detalhes de javadoc
